.circleci/modernize/modernize.py

```diff
@@ -23,6 +23,10 @@
 #   -j --processes <jobs count> run concurrently.
 # Note: This is only python3 script.
 
+if False:
```

Comment:
Do you need to leave this in? If so, please add a comment explain what it is for.

Reply:
Yeah, sorry, we have a comment in other places where type annotations are used, but not here (will also add it there).

In short, that's a workaround for Python 2 where ``typing`` package is not available by default. 

When MyPy runs under Python 3, those lines will still be executed / parsed, but not during a regular program run.